Beth README.md

```diff
@@ -0,0 +1,29 @@
+## Chitter Challenge
+
+Solo project Makers Friday 15 Oct 2021
+
+https://github.com/makersacademy/chitter-challenge-apprenticeships
+
+
+We are going to write a small Twitter clone that will allow the users to post messages to a public stream.
+
+## Using Ruby Web project setup for guidance
+
+https://github.com/makersacademy/course/blob/main/pills/ruby_web_project_setup_list.md
+
+## Database setup
+
+
+## User stories
+```
+As a Maker
+So that I can see what people are doing
+I want to see all the messages (peeps)
+in a browser
+```
+
+```
+As a Maker
+So that I can let people know what I am doing  
+I want to post a message (peep) to chitter
+```
```

Comment:
It would be helpful to see a plan, or just a couple of sentences sharing your thinking/approach to these stories. That would help give context to the code, making it more straightforward to review.

Reply:
Adding to this, it would help to see what you were planning to do next.